kuma/landing/jinja2/landing/includes/contribution-form.html

```diff
@@ -0,0 +1,52 @@
+<div class="contribution-form contribution-banner">
+    <div class="row">
+        <img src="{{ static('img/hero-dino-blank.png') }}" class="backdrop-image" alt="">
+        <div class="column">
+            <div class="container">
+                <h2 class="highlight highlight-spanned">
+                    <span class="highlight-span">{{ _('Help Support MDN for as little as $2')}}</span>
+                </h2>
+                <h4>{{_('We need your help to grow to the next stage.')}}</h4>
+                <p class="body">{{_('Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud.')}}</p>
+                {% if is_cta %}
+                <div class="button-group form-footer">
+                    <a href="#" class="cta-link">
+                        {{_('Learn More')}} {% include 'includes/icons/arrows/arrow.svg' %}
+                    </a>
+                    <a href="#" class="cta-link">
+                        {{_('Feedback')}} {% include 'includes/icons/arrows/arrow.svg' %}
+                    </a>
+                </div>
+                {% endif %}
+            </div>
+        </div>
+        <div class="column">
+            <form id="contribute-form-2" class="container" method="post" action="">
+                {% csrf_token %}
+                <div class="form-group">
+                    {% for field in form %}
+                        {% if field.label and not field.hidden %}
+                            <span class="label"> {{field.label}} </span>
+                        {% endif %}
+                        {{ field }}
+                        {{ field.errors }}
+                    {% endfor %}
+                </div>
+                <div class="form-footer">
+                    <div class="payment-details">
+                        <img src="{{ static('img/mc-logo.png') }}" alt="mastercard">
+                        <img src="{{ static('img/visa-logo.png') }}" alt="visa">
+                        <img src="{{ static('img/ae-logo.png.png') }}" alt="american express">
+                    </div>
+                    <button type="button" id="stripe_submit">
+                        <div class="payment-details">
```

Comment:
Is the `<div class="payment-details">` duplicated? It appears to include the images a second time.

Reply:
Its there for mobile view, we're putting the payment options in the button due to space constraints. Though maybe this is something we can do differently.